data-prepper-plugins/kafka-connect-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafkaconnect/configuration/ConnectorConfig.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.kafkaconnect.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.opensearch.dataprepper.plugins.kafkaconnect.util.Connector;
+
+import java.util.List;
+import java.util.Properties;
+
+public abstract class ConnectorConfig {
+    @JsonProperty("force_update")
+    public Boolean forceUpdate = false;
+    private String bootstrapServers;
```

Comment:
Should this be a `List<String>`?

Reply:
It is String, this config is not used for user input, it is designed for Kafka Connect's worker property, and it is string.
For customer's input, it is List<String>, and I have logic to validate and turn it into string.